rpcwebsocket.go

```diff
@@ -228,6 +230,20 @@ func (m *wsNotificationManager) NotifyBlockDisconnected(block *dcrutil.Block) {
 	}
 }
 
+// NotifyVotes passes a block newly-connected to the best chain
+// to the notification manager for block and transaction notification
+// processing.
+func (m *wsNotificationManager) NotifyBlockConnected(block *dcrutil.Block) {
```

Comment:
copy/paste error?  or not yet implemented?

    NotifyBlockConnected vs. NotifyVotes